kafka-rest/src/main/java/io/confluent/kafkarest/resources/v3/ProducerMetrics.java

```diff
@@ -94,25 +101,26 @@ final class ProducerMetrics {
   static final String REQUEST_LATENCY_PCT_METRIC_PREFIX = "request-latency-";
   private static final String REQUEST_LATENCY_PCT_METRIC_DOC = "Request latency percentiles.";
 
-  public static final long EXPIRY_SECONDS = TimeUnit.DAYS.toSeconds(7);
-
   private final Metrics metrics;
   private final String jmxPrefix;
   private final String requestSensorName;
   private final String requestSizeSensorName;
   private final String responseSensorName;
   private final String recordErrorSensorName;
+  private final String recordRateLimitedSensorName;
   private final String requestLatencySensorName;
 
-  ProducerMetrics(KafkaRestConfig config, Time time) {
-    this(config, time, ImmutableMap.of());
+  // TODO: Used in ce-kafka-rest. To be deleted.
```

Comment:
nit: I personally think we should not burden the open-source code with details related to the internal Confluent source code. Maybe reword to something like "Not considered a public API, to be deleted once known usages are removed."

Reply:
Done.